midend/simplifyKey.h

```diff
@@ -86,7 +120,13 @@ class DoSimplifyKey : public Transform {
     const IR::Node* postorder(IR::P4Table* table) override;
 };
 
-// Use a policy to decide when a key expression is too complex.
+/**
```

Comment:
This is incorrect, this pass actually simplifies the keys, it does not determine anything.
You deleted the original comment, which indicated what the policy parameter is about; that should become a `@param` comment.